nav_msgs/Odometry pose
---
# Return borrowed from std_srvs/Trigger
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages